prExplorer/client-app/src/features/products/dashboard/ProductList.tsx

```diff
@@ -1,44 +1,139 @@
-                <Label basic content={product.category} />
-              </Item.Extra>
-            </Item.Content>
-          </Item>
-        ))}
-      </Item.Group>
+      <Input
+        placeholder="Search..."
+        value={searchTerm}
+        onChange={(e) => setSearchTerm(e.target.value)}
+        action={{
+          icon: "search",
+          onClick: () => setSearchTerm(""),
+        }}
+      />
+      <Table sortable celled>
+        <Table.Header>
+          <Table.Row>
+            <Table.HeaderCell
+              sorted={column === "title" ? direction : undefined}
+              onClick={handleSort("title")}
+            >
+              Name
+            </Table.HeaderCell>
+            <Table.HeaderCell
+              sorted={column === "price" ? direction : undefined}
+              onClick={handleSort("price")}
+            >
+              Price
+            </Table.HeaderCell>
+            <Table.HeaderCell
+              sorted={column === "discountPercentage" ? direction : undefined}
+              onClick={handleSort("discountPercentage")}
+            >
+              Discount
+            </Table.HeaderCell>
+            <Table.HeaderCell
+              sorted={column === "rating" ? direction : undefined}
+              onClick={handleSort("rating")}
+            >
+              Rating
+            </Table.HeaderCell>
+            <Table.HeaderCell
+              sorted={column === "stock" ? direction : undefined}
+              onClick={handleSort("stock")}
+            >
+              Stock
+            </Table.HeaderCell>
+            <Table.HeaderCell
+              sorted={column === "category" ? direction : undefined}
+              onClick={handleSort("category")}
+            >
+              Category
+            </Table.HeaderCell>
+            <Table.HeaderCell>Thumbnail</Table.HeaderCell> {/* Yeni sütun */}
+            <Table.HeaderCell>View</Table.HeaderCell>
+            <Table.HeaderCell>Delete</Table.HeaderCell>
+          </Table.Row>
+        </Table.Header>
+        <Table.Body>
+          {filteredProducts.map((product) => (
+            <Table.Row key={product.id}>
+              <Table.Cell>{product.title}</Table.Cell>
+              <Table.Cell>{product.price}</Table.Cell>
+              <Table.Cell>{product.discountPercentage}</Table.Cell>
+              <Table.Cell>{product.rating}</Table.Cell>
+              <Table.Cell>{product.stock}</Table.Cell>
+              <Table.Cell>{product.category}</Table.Cell>
+              <Table.Cell style={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+                <Image src={product.thumbnail} size="mini" circular style={{ width: '50px', height: '50px', borderRadius: '50%', objectFit: 'cover' }} /> {/* Thumbnail gösterme */}
+              </Table.Cell>
+              <Table.Cell>
+                <Button
```

Comment:
You can make these "View" and "Delete" buttons a separate components, so you can reuse them on every list/table you will create in the future